content/livesync/connector/index.textile

```diff
@@ -87,36 +97,37 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
 The @docker-compose.yml@ will load this configuration into the @adbc@ container as environment variables.
 
 <aside data-type='important'>
-<p>If using your own API key, ensure that it has the @publish@ "capability":/auth/capabilities on all channels that you intend to publish to.</p>
+<p>If using an API key from a provider other than Ably, verify it has the @publish@ "capability":/auth/capabilities on all channels you intend to publish to. </p>
```

Comment:
This isn't quite right, only Ably can issue API keys to access Ably. Perhaps just "Ensure your API key has the publish capability ..."

Reply:
👍